docs/source/Tutorials/custom.rst

```diff
@@ -110,6 +110,76 @@ known good output of "Hello World!".
 
 .. todo:: Add (or link to) more detailed examples of bare-metal and rocc workloads.
 
+
+Using the same directory structure for both bare-metal and buildroot
```

Comment:
Looking at the other tutorials here (https://firemarshal.readthedocs.io/en/latest/Tutorials/custom.html), it looks like we really just need to add one on BuildRoot and not baremetal. Even then, the Fedora tutorial describes much of what you want... so I think if the docs really want to get fixed I would do the following:

1. Re-order the "Custom Workloads" tutorials to be 1. BuildRoot Linux Hello World (w/ descriptions of parameters probably taken from the existing Fedora tutorial), 2. Baremetal Hello World 3. Fedora Hello World. 

2. For sharing, I think the `test/bare.yaml` workload already implements the baremetal hello world... instead of copying that we should repurpose it for the tutorial (I think that workload compile could be simplified). OR we can symlink the example from the test area (or vice version).